### ISSUE TITLE
Wrong height of QR-Bill for PDF invoices?

### ISSUE THREAD

**Issue by kohlerdominik** [sprain/php-swiss-qr-bill#74]
Hi @sprain 

Maybe you can verify or deny this issue, as I'm a little in doubt right now.

I interpret from the [style guidelines](https://www.paymentstandards.ch/dam/downloads/style-guide-en.pdf) (page 6), that the qr-bill-part should be 105mm heigh, no matter if it's a paper printed invoice or a pdf-invoice.

But right now it seems that the line `Separate before paying in` is put in at the top within the 105mm `#qr-bill` div, without compensation of height, meaning that the separation line is moved 12px down, or in other words, the separated part has a height of `105mm-12px` (for me in firefox, might be interpreted differently in other browsers)

**Comment by sprain** [comment]
> But right now it seems that the line Separate before paying in is put in at the top within the 105mm #qr-bill div, without compensation of height, meaning that the separation line is moved 12px down

I had a look and I think I see what's going on:

* The separate-before-payment information is indeed placed within the div that defines the height of 105mm.
* But because all other elements (like title section, information section, amount section, acceptance section and padding) also have their measurements defined, that div including the separate-payment-information actually becomes higher than 105mm – and therefore the outputted result is correct and matches the guidelines.

In conclusion, the definition `#qr-bill{height: 105mm;}` could be removed as it's actually doing nothing.

Would you agree with this?

**Comment by kohlerdominik** [comment]
Ah, i didn't even notice. The separate-row has fixed height no matter if with or without content.

Yes, in this case i think your proposal is the correct solution. Let the the height be built from the inner elements.
